_contents/S0-L05.md

```diff
@@ -42,4 +42,416 @@ In this session, our readings cover:
+
+RL Training: The objective function considers the rewards from the RM model output and includes a KL penalty to prevent substantial deviations from the pre-trained model.
+![image](https://github.com/suuuuu017/2024sp-GenAI-Risk-Benefits/assets/69375710/c06ae93f-3e2a-4f3f-a454-9b766d99ab8b)
+The reward from RM, coupled with Proximal Policy Optimization (PPO) in the RL phase, ensures the model is fine-tuned to customer prompts effectively. KL-penalty in the RL training phase prevents the model's policy from diverging too far from the original pre-trained model, thus maintaining performance consistency.
+
+
+#### Preference Model
+![917af754ce4166be556529a69547c1a](https://github.com/FeilianDai/2024sp-GenAI-Risk-Benefits/assets/74888568/c4ae698e-5fe0-4992-b77a-37e90b290119)
+
+GPT-3 when it is provided a few-shot prefix to ‘prompt’ it into an instruction-following
+mode (GPT-3-prompted)
+- PPO always above 0.5
+- 1.3B PPO is better than 175B SFT
+
+#### Meta-result for API distribution
+![6e97b8b25df8105421ecacb78ec2722](https://github.com/FeilianDai/2024sp-GenAI-Risk-Benefits/assets/74888568/fa20df1c-8125-4ac7-b4a3-11b341006b8c)
+
+#### Likert Comparison between FLAN and T0
+![1ee831b6e1f0d4adfca1eb09264efc1](https://github.com/FeilianDai/2024sp-GenAI-Risk-Benefits/assets/74888568/1bff9df2-93d5-4ef9-aaad-e9145f58695d)
+
+#### Results on TruthfulQA dataset
+![26a2b9fc26bb7e3510694b63747746a](https://github.com/FeilianDai/2024sp-GenAI-Risk-Benefits/assets/74888568/dbb08818-6e1c-48d0-b7a8-1ebf88749cc4)
+
+#### Reliable Answering
+![079f2725f2de7dc1b068e00a788c510](https://github.com/FeilianDai/2024sp-GenAI-Risk-Benefits/assets/74888568/2c61f063-5f34-4470-beff-9275bf8e6c2f)
+
+#### Simple Mistakes
+![6e281edf7b757b83d9075b303c60ade](https://github.com/FeilianDai/2024sp-GenAI-Risk-Benefits/assets/74888568/2dce4cff-995e-40bc-8aea-7dd2aa40072d)
+
+#### Summary & Discussions
+1. Demonstrate that this alignment technique can align to a specific human reference group for a specific application
+2. Implication
+    - Cost effective than training larger model
+    - More research is needed for generalization
+
+### Direct Preference Optimization: Your Language Model is Secretly a Reward Model
+
+#### RLFH Recap
+![0096b71ca7f67677e69d938da4771a9](https://github.com/FeilianDai/2024sp-GenAI-Risk-Benefits/assets/74888568/0b749f5a-43ef-4beb-92b0-ad631d3df1a1)
+
+1. RLHF typically begins by fine-tuning a pre-trained LM with supervised learning on high-quality data for the downstream task(s) of interest (dialogue, summarization, etc.), to obtain a model πSFT.
+
+2. Reward Modelling Phase: In the second phase the SFT model is prompted with prompts x to produce pairs of answers (y1, y2) ∼ πSFT(y | x). These are then presented to human labelers who express preferences for one answer, denoted as yw ≻ yl | x where yw and yl denotes the preferred and dispreferred completion amongst (y1, y2) respectively. 
```

Comment:
After compiling, this paragraph generates a table? I assume it is not expected. I think it is because of the math editing. Please use Markdown for math editing.